src/Controller/EnMarche/CitizenProjectController.php

```diff
@@ -45,4 +49,35 @@ public function helpAction(): Response
 
         return new Response();
     }
+
+    /**
+     * @Route("/skills/autocompletion",
+     *     name="app_citizen_project_skills_autocomplete",
+     *     condition="request.isXmlHttpRequest()"
+     * )
+     * @Method("GET")
+     * @Security("is_granted('IS_AUTHENTICATED_FULLY')")
+     */
+    public function skillsAutocompleteAction(Request $request)
+    {
+        if (!$category = $this->getDoctrine()->getRepository(CitizenProjectCategory::class)->find($request->query->get('category'))) {
+            return new JsonResponse([], Response::HTTP_NOT_FOUND);
+        }
+
+        $citizenProjectCategorySkills = $this
+            ->getDoctrine()
+            ->getRepository(CitizenProjectCategorySkill::class)
+            ->findByCitizenProjectCategoryAndTerm($category, $request->query->get('term', ''));
+
+        $result = [];
```

Comment:
`$results` and no need for the initialization use `return $results ?? [];` (this also compiles better optimized C code).